src/sentry/web/frontend/debug/mail.py

```diff
@@ -511,6 +509,47 @@ def digest(request):
             state["event_counts"][group.id] = random.randint(10, 1e4)
             state["user_counts"][group.id] = random.randint(10, 1e4)
 
+    # add in performance issues
+    for _ in range(random.randint(1, 10)):
+        with override_options(
+            {
+                "performance.issues.all.problem-creation": 1.0,
+                "performance.issues.all.problem-detection": 1.0,
+                "performance.issues.n_plus_one_db.problem-creation": 1.0,
+            }
+        ):
+            # make this consistent for acceptance tests
+            perf_data = dict(
+                load_data(
+                    "transaction-n-plus-one",
+                    timestamp=datetime(2022, 11, 11, 21, 39, 23, 30723),
+                )
+            )
+            perf_data["event_id"] = "44f1419e73884cd2b45c79918f4b6dc4"
+            perf_event_manager = EventManager(perf_data)
+            perf_event_manager.normalize()
+            perf_data = perf_event_manager.get_data()
+            perf_event = perf_event_manager.save(project.id)
+
+        perf_event = perf_event.for_group(perf_event.groups[0])
+        perf_group = perf_event.group
```

Comment:
Do we need to hardcode the group id here?

Reply:
Probably yes, updating the acceptance test now